src/app/screens/TimesheetScreen/component/employeeCard.tsx

```diff
@@ -5,19 +5,29 @@ import Touchable from '../../../components/touchable';
 import Typography from '../../../components/typography';
 
 import {navigate} from '../../../navigation';
+import {dateFormate} from '../../../utils/date';
 
 import {Arrow} from '../../../constant/icons';
 import {USER_TIMESHEET} from '../../../constant/screenNames';
+import {ISO_DATE_FROMAT} from '../../../constant/date';
 
 type Props = {
   name: string;
   email: string;
   userId: string;
+  startDate: Date;
+  endDate: Date;
 };
 
-const EmployeeCard = ({name, email, userId}: Props) => {
+const EmployeeCard = ({name, email, userId, startDate, endDate}: Props) => {
   const handleNavigation = () =>
-    navigate(USER_TIMESHEET, {name, email, user_id: userId});
+    navigate(USER_TIMESHEET, {
+      name,
+      email,
+      user_id: userId,
+      startDate: dateFormate(startDate, ISO_DATE_FROMAT),
+      endDate: dateFormate(endDate, ISO_DATE_FROMAT),
```

Comment:
why convert it to string? Can we pass directly startDate and endDate?

Reply:
Params should be serialisable. Date is not a serialisable object so we are passing ISO format string instead.